base/applications/mspaint/winproc.cpp

```diff
@@ -584,6 +584,9 @@ ULONG CMainWindow::OpenMailerWithAttachmentW(LPCWSTR pszPathName)
     CStringW strFileTitle;
     if (PathFileExistsW(pszPathName)) // File does exist?
     {
+        if (!ConfirmSave())
```

Comment:
I've checked with Windows' own MSPaint, and when the file exists, but has been modified and not yet saved, and one tries to send it by mail, there is *_NO_* prompt for save confirmation, *but* the file with modifications is nevertheless attached: a temporary file is created.

Reply:
Always temp file?

Reply:
According to observations, it appears it's always, yes, **UNLESS** the file hasn't been modified (or has been manually saved just prior to sending it).

Reply:
Now FileType is fixed.

Reply:
Now file title is fixed.